src/py/mat3ra/notebooks/utils/core/json.py

```
@@ -0,0 +1,50 @@
+
+        with open(os.path.join(web_dir, "renderjson.css")) as fp:
+            css = fp.read()
+
+        with open(os.path.join(web_dir, "renderjson.js")) as fp:
+            js = fp.read()
+
+        display(HTML(f'<style>{css}</style><div id="{id}"></div>'))
+        display(
+            HTML(
+                f"<script>{js} "
+                f"renderjson.set_show_to_level({str(level)}); "
```

Ruff (0.15.12)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)